### PR TITLE
add `unitt` package

### PR DESCRIPTION
**Pull request by RickBarretto**
# 📖 Description

Unitt is a basic unit-test tool for the Arturo Programming language.

## 📦 New package

- [x] I have updated the [packages/list.art](https://github.com/arturo-lang/pkgr.art/blob/main/packages/list.art):
    * All you have to do is add an entry like:
       ```red
       packageName: "owner/repo"
       ```
- [x] The repo in question is formatted as a valid Arturo package:
    * either have a `main.art` file and/or an `info.art` file specifying the entry point, `depends` and `requires`
- [x] There is at least one published release:
    * Arturo's package manager versioning uses the published releases' tags which have to conform to SemVer

## 🗒 Notes
* My first publish is marked as pre-release
* I'm not using `main.art`, but `unitt.art`, declared in the entry.

My current `info.art`:

```art
entry: {unitt}
depends: []
requires: 0.9.83
```

### CODE REVIEW

**Comment by drkameleon** [comment]
First of all: awesome work + extremely important so that we start seeing how the packager works. 🚀 

So, first, there is a Verification stage (if this passes, everything is gonna be ok later on) - this is what we want when a new PR is made. (And then, once merged, it's deployed to the server via the CI action). So, all great...

Now, the Verification stage here initially failed (so that you have a good idea) because:
- the validator is written in Arturo
- so we need to have Arturo itself set up and installed
- for that I'm using our [GitHub Action (for Arturo)](https://github.com/arturo-lang/arturo-action)
- the action was using the old `./build.nims` to build & install Arturo
- so, basically it couldn't get Arturo properly installed

*That* was fixed and I re-run the Verification stage on demand. And all set - with that.

Now, I'm seeing another error:

Screenshot 2024-01-29 at 10 58 09

Which is informative enough, but we'd have to see what exactly it is that cause that problem (my guess would be your `-pre` suffix for the release name - obviously it's a perfectly valid SemVer string, so it *should* be recognized).

Let's see... Time for debugging! :)

**Comment by drkameleon** [comment]
So, I *think* I've figured it out:

basically, it wasn't able to properly identify your release (I had it to look for the release **name** - which in your case was "First pre-release" - while it should probably be looking for the **name_tag** - which is v0.1.1-pre).

Let's see... Testing it locally - at least - it seems to be working! :)

**Comment by drkameleon** [comment]
Re-running CI tests 🎯

**Comment by drkameleon** [comment]
And success - finally! 🎉 

Screenshot 2024-01-29 at 11 46 32

Now, I'll merge and supposedly everything will work fine... 🥁 🥁 🥁 

Let's see... That's a crucial moment! 🚀

**Review by drkameleon — APPROVED** [review]

**Comment by RickBarretto** [comment]
🙂, interesting that with this push you've discovered more two hidden bugs. (pre-suffix and looking at name instead of tags).

I'm excited to test this!
